rcl/int32_publisher/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>int32_publisher_rcl</name>
+  <version>0.0.1</version>
+  <description>Example of a publisher using int32</description>
+  <maintainer email="julianbermudez@eprosima.com">Julian Bermudez</maintainer>
+  <license>Apache License 2.0</license>
```

Comment:
```suggestion
  <license>Apache-2.0</license>
```
Cf. previous comment and https://spdx.org/licenses/

Reply:
Thanks for the information. Change!